freshmaker/pyxis_gql.py

```diff
@@ -496,3 +496,65 @@ def find_images_by_names(self, names):
             page_num += 1
 
         return images
+
+    def find_published_images_by_name_version_content_sets(self, name, version, content_sets):
```

Comment:
Can we rename it to `find_images_by_name_version` and
`find_images_by_name_version(self, name, version, published=True, content_sets=None)`?